packages/components/psammead-brand/src/index.stories.jsx

```diff
@@ -118,3 +122,5 @@ storiesOf('Components|Brand', module)
     },
     { notes },
   );
+
+buildRTLSubstories(STORY_KIND, { include: ['with brand link'] });
```

Comment:
Would it be possible to set other props to default to a RTL service? e.g. the brand currently uses the News SVG by default. It'd be good if it could use the Arabic SVG along with the arabic service prop. 

<img width="1434" alt="Screenshot brand example" src="https://user-images.githubusercontent.com/3028997/67414716-8d4f8880-f5bb-11e9-8269-9ee5ec5411d4.png">

Reply:
Nice idea. I don't know why this didn't cross my mind. In order to do this I removed the uppercased service options, which were for presentational purposes anyway, so that I didn't have to add more complexity to these stories. I hope that's ok.

Reply:
That's perfect, thanks! I like that the SVG now defaults to Arabic here now. 
<img width="1440" alt="Screenshot of the RTL Brand story" src="https://user-images.githubusercontent.com/3028997/67466321-c41ab280-f63e-11e9-9b80-b826af8a1c64.png">